.gitattributes

```diff
@@ -1 +1,2 @@
 khmer/_version.py export-subst
+ChangeLog    merge=merge-changelog
```

Comment:
Thanks for catching that. Probably shouldn't be merged as not everyone will have git-merge-changelog installed. I've moved it to my personal site-wide configuration file.